sumologic/sumologic_cse_automation.go

```diff
@@ -0,0 +1,98 @@
+package sumologic
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func (s *Client) GetCSEAutomation(id string) (*CSEAutomation, error) {
+	data, _, err := s.Get(fmt.Sprintf("sec/v1/automations/%s", id))
+	if err != nil {
+		return nil, err
+	}
+
+	if data == nil {
+		return nil, nil
+	}
+
+	var response CSEAutomationResponse
+	err = json.Unmarshal(data, &response)
+	if err != nil {
+		return nil, err
+	}
+
+	return &response.CSEAutomation, nil
+}
+
+func (s *Client) DeleteCSEAutomation(id string) error {
+	_, err := s.Delete(fmt.Sprintf("sec/v1/automations/%s", id))
+
+	return err
+}
+
+func (s *Client) CreateCSEAutomation(CSEAutomation CSEAutomation) (string, error) {
+
+	request := CSEAutomationRequestPost{
+		CSEAutomation: CSEAutomation,
+	}
+
+	var response CSEAutomationResponse
```

Comment:
Minor nit, this could be right above line 46 for slightly improved clarity, my brain was like "why is this here?" until I read the next 5 lines.

Reply:
e.g.
```
var response CSEAutomationResponse
err = json.Unmarshal(responseBody, &response)
```